flight/Modules/UAVOFrSKYSensorHubBridge/UAVOFrSKYSensorHubBridge.c

```diff
@@ -178,15 +178,15 @@ static const uint8_t frsky_rates[] = {
 // ****************
 // Private variables
 
-static struct pios_thread *uavoFrSKYSensorHubBridgeTaskHandle;
+static struct {
+	struct pios_thread *task_handle;
 
-static uint32_t frsky_port;
+	uint32_t frsky_port;
 
-static bool module_enabled;
+	uint8_t frame_ticks[MAXSTREAMS];
 
-static uint8_t *frame_ticks;
-
-static uint8_t *serial_buf;
+	uint8_t serial_buf[FRSKY_MAX_PACKET_LEN];
+} *frsky_state;
```

Comment:
It's helpful if they have unique names because they're easier to get to with a debugger.  So the intention here is the state information for the frsky module (of course, we have 2.5 frsky modules heh).

I could call it shub_global.  If this sounds good I'll try and get this done before I'm away for a few days.